pages/_app.tsx

```diff
@@ -0,0 +1,5 @@
+import "../src/client/styles/main.scss";
```

Comment:
The file name and the `MyApp` name is all convention. I introduced it because I wanted to add the global scss file to the app.